### PR TITLE
create a pug page payment-order

### PR DESCRIPTION
**Pull request by AnastasiiaKolodiazhna**
Created a pug page payment-order.

### CODE REVIEW

**Comment by Kiyooooko** [comment]
fix bug pls

**Comment by Kiyooooko** [comment]
CI didn't run correctly

File: email-templates/payment-order.pug [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    div.contentPaymentText
+        div.productTableContent
+            p.congrats Вітаємо! #[br] Залшився ще один крок для отримання Вашого замовлення: #[br]
+            table.productTable 
+                tr.tableHead
+                    th Фото
+                    th Назва
+                    th Кількість
+                    th Ціна
+                each val in cart.items
```
Comment (Kiyooooko) [inline email-templates/payment-order.pug:14]: you have to add if...else for product prom constructor

**Comment by Kiyooooko** [comment]
Why you don't use our PR template???????
Screenshot 2021-03-24 at 10 42 31

**Comment by Kiyooooko** [comment]
Why this fields is empty?
Screenshot 2021-03-24 at 10 42 58

File: email-templates/index.js [hunk 2]
```diff
@@ -33,6 +33,10 @@ const htmlTemplates = {
     subject: '[HORONDI] Invitation to become an admin',
     templateFileName: 'confirm-admin-email',
   },
+  [EmailActions.PAYMENT_ORDER]: {
+    subject: '[HORONDI] Pay for your order',
+    templateFileName: 'payment-order',
```
Comment (grebiniuk) [inline email-templates/index.js:38]: should be "order-peyment", "payment order" is a completely different thing.

**Review by grebiniuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**